clients/java/signalr/src/main/java/com/microsoft/signalr/Subscription.java

```diff
@@ -5,17 +5,23 @@
 
 import java.util.List;
 
+/**
+ * A type that represents being subscribed to invocations to a specific target.
```

Comment:
Suggest: Represents the registration of a handler for a client method.

"A type" is self-evident, since it's a class, and we don't use prefixes like that in other docs. "Represents being subscribed" feels a little awkward, and the type really represents the registration.

I kinda wonder if we should rename this type since a) it might conflict/confuse with [Rx's `Subscription`](http://reactivex.io/RxJava/javadoc/rx/Subscription.html) and b) we don't really use the term "subscribe" when you register the handler. It's really more like "defining" a method. We can talk about it post-preview3 (I know it's public API but it's minor and I don't think it's worth trying to squeeze it in; I'd rather just not rename it than try to do that)

Reply:
Yeah, this was a weird one.